packages/terra-button-group/src/terra-dev-site/doc/button-group/ButtonGroup.1.doc.mdx

```diff
@@ -1,3 +1,16 @@
+import { Badge } from 'terra-button-group/package.json?dev-site-package';
+
+import ButtonGroupWithIcons from '../example/ButtonGroupWithIcons?dev-site-example';
```

Comment:
These are fine as they are. I wouldn't use the `terra-button-group` alias here as it would make the import longer.